bundles/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/protocol/SSO/AuthorizationCodeExchangeResponse.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.tesla.internal.protocol.SSO;
```

Comment:
Please stick to lower case package names.
```suggestion
package org.openhab.binding.tesla.internal.protocol.sso;
```